docs/platform/object-storage/object-storage-use-cases/index.md

```diff
@@ -55,14 +55,17 @@ Big Data typically describes data sets that are so large and so diverse that it
 
 ### Artifact Storage
 
-As more and more of the development life cycle becomes automated and tested, more and more artifacts are generated in the process. Object Storage is a great solution for developers looking to store all these artifacts, such as the bulk collection of logs. Sharing these artifacts is as simple as as sharing a URL. If you'd rather these logs stay private, you can also distribute an access key for them.
+As more and more of the development life cycle becomes automated and tested, more and more artifacts are generated in the process. Object Storage is a great solution for developers looking to store all these artifacts, such as the bulk collection of logs. Sharing these artifacts is as simple as sharing a URL. If you'd rather these logs stay private, you can also distribute an access key for them.
 
 ### Cold Storage
 
-Object Storage is significantly cheaper than Block Storage. While Object Storage does incur a cost when retrieving data, the cost benefit for infrequently accessed data is something that should not be glossed over. As well, the outbound data transfer for Linode Object Storage is part of your Linode account's [total transfer pool](/docs/platform/billing-and-support/network-transfer-quota/), which can further reduce costs if you are running Linode instances too.
+Object Storage is in the majority of cases significantly cheaper than Block Storage. While Object Storage can incur a cost when retrieving data, the cost benefit for infrequently accessed data can provide you with an overall cost reduction when compared to similar methods.
 
 Similarly, Object Storage has benefits over [tape storage](https://en.wikipedia.org/wiki/Tape_drive). Tape storage is frequently used for archival purposes, but the read times that come with tape storage are many times more than what you'll find with Object Storage. Special considerations have to be made when transferring tape drive data, such as the ability to ship drives safely across a country. With Object Storage, this data is available through HTTP from anywhere in the world.
 
+{{< note >}}
+The outbound data transfer for Linode Object Storage is part of your Linode account's [total transfer pool](/docs/platform/billing-and-support/network-transfer-quota/), which will reduce or completely eliminate transfer costs for object storage if you are running Linode instances too. When you have expended your allotted transfer pool, you will be billed at a rate of $0.02 per GB for outbound transfers.
+{{< /note >}}
```

Comment:
Adjusted the information on billing to clarify what you will and won't be billed for. With enough of a transfer pool, you won't be billed for transfer usage at all